docs/permissions/plugin-authors/05-frontend-authorization.md

```diff
@@ -199,7 +199,7 @@ const routes = (
       <RequirePermission permission={todoListCreatePermission}>
         <TodoListPage />
       </RequirePermission>
-      {/* highlight-add-end */}}
+      {/* highlight-add-end */}
```

Comment:
@benjdlambert, @Rugvip - I think we need to bring that back. It's the closing brace and angle-bracket for the `element` prop and `Route`, respectively, on line 196:
```suggestion
      {/* highlight-add-end */}
    }>
```

Reply:
Ah yep, found it :+1: